clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -566,6 +566,49 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+template <size_t GROUP_WORK_ITEMS, typename InputT, int ITEMS_PER_THREAD,
```

Comment:
Similar to `GROUP_WORK_ITEMS`, I think `ITEMS_PER_THREAD` should be `ITEMS_PER_WORK_ITEM`, so we stick with SYCL terminology.

Reply:
Yes added